### PR TITLE
cleanup and define all unknowns in zone reading

### PR DESCRIPTION
**Pull request by aronwk-aaron**
fix for movement path config reading
Tested that this has no functional changes. There may can be changes made since this data is more clearly defined.
This is also in prep for npc movement pathing
reference: https://github.com/lcdr/lu_formats/blob/master/files/luz.ksy

### CODE REVIEW

File: dZoneManager/Zone.cpp [hunk 1]
```diff
@@ -403,10 +389,17 @@ void Zone::LoadPath(std::istream& file) {
 			}
 		}
 	} else if (path.pathType == PathType::Property) {
-		int32_t unknown;
-		BinaryIO::BinaryRead(file, unknown);
+
+		int32_t propertyPathType;
+		BinaryIO::BinaryRead(file, propertyPathType);
+		path.property.pathType = PropertyPathType(propertyPathType);
```
Comment (EmosewaMC) [inline dZoneManager/Zone.cpp:395]: I believe we could shorten this to BinaryIO::BinaryRead(file, path.property.pathType);
Does this work here?

File: dZoneManager/Zone.cpp [hunk 2]
```diff
@@ -403,10 +389,17 @@ void Zone::LoadPath(std::istream& file) {
...
+
 		BinaryIO::BinaryRead(file, path.property.price);
-		BinaryIO::BinaryRead(file, path.property.rentalTime);
+
+		int32_t rentalTime;
+		BinaryIO::BinaryRead(file, rentalTime);
+		path.property.rentalTime = PropertyRentalTimeUnit(rentalTime);
```
Comment (EmosewaMC) [inline dZoneManager/Zone.cpp:401]: Same as above

**Review by EmosewaMC — COMMENTED** [review]
Couple questions so far

File: dZoneManager/Zone.cpp (hunk 1 above)
Comment (aronwk-aaron) [reply dZoneManager/Zone.cpp:395]: ahhh you're correct, was under the impression that due to the enum class that you have to declare it like this

**Review by EmosewaMC — APPROVED** [review]
small small comment

**Review by EmosewaMC — APPROVED** [review]

**Review by Jettford — APPROVED** [review]
